system-x/services/db/common/src/main/java/software/tnb/db/common/openshift/OpenshiftDB.java

```diff
@@ -96,6 +112,8 @@ public void create() {
 
     @Override
     public void undeploy() {
+        SecurityContextConstraints scc = OpenshiftClient.get().securityContextConstraints().withName(SCC_NAME).edit();
```

Comment:
I think the `.edit()` doesn't do anything, perhaps you wanted `.get()`

Reply:
sure, this already does something - changed, thanks!